### DISCUSSION TITLE
Feature requests

### DISCUSSION THREAD

**Discussion by RahmanM** [arnoudkooi/SN-Utils#314] [Q&A] [answered]
He there,

A couple of feature requests if possible at all!

1. Open the page back to backend with Searchbar etc e.g. I am opening the list using incident.LIST command which will open a new whindow. If I press a button combination then should reopen the same page in backned. Like chaning the URL from https://<>.service-now.com/incident_list.do to https://<>.service-now.com/now/nav/ui/classic/params/target/incident_list.do. I was actually using a separate Chrome Extenstion (ServiceNow Frameizer) but Chrome removed the extension as it was violating the t&c.
2. Ability to populate a Catalog item or any form with some sample data. This will really help when testing some big forms with many required fields and you have to submit the same form many times.

Many thanks

Rahman

**Comment by arnoudkooi** [comment, marked as answer]
1. Have a look at [slashcommands](https://www.arnoudkooi.com/cheatsheet/). 
The `/pop` command will get you in out the frame, you can assign a browser shortcut to this as well (by default CTRL-2)
The command `/incident -pi` would achieve what you want with a single slashcommand

2. Try the slashcommand `/rnd`  

Be sure to watch my videos and read my twiiter feed to get the most out of SN Utils,[ links on my website](https://www.arnoudkooi.com/)

**Reply by RahmanM** [reply]
Hi,

Thank you so much for the quick response!

/rnd is perfect and that is exactly what I needed!

/pop helps with what I needed but possibly a keyboard combination would have helped. CTRL-2 is not working for me. I have windows machine. Can you please clarify on how I can assign a browser short-cut for /pop? My version /pop is greyed out and does not allow changing it.

Many thanks

**Reply by arnoudkooi** [reply]
<img width="1017" alt="image" src="https://user-images.githubusercontent.com/840781/196622530-d6b1a127-4400-44d1-b73d-87018dbe3326.png">
